### DISCUSSION TITLE
How to port old cache-access

### DISCUSSION THREAD

**Discussion by christ0phd** [pelicanmapping/osgearth#2036] [Support Q&A]
Hi all,
it's been a while where my app was propper running using osgEarth 2.9. But currently I have to port the whole stuff to the youngest release. The codebase is allready building but for the preparation I need to port the former implementation of the cache seeding. As far as I can see, the old `osgearth_cache` tool is gone. Trying to build the 'osgearth_seed' tool ended up in several compiler errors.
Please, @gwaldron, is there any recommended way on how to prepare local caches?

Christoph

**Comment by gwaldron** [comment]
We removed the utility many moons ago because it was problematic for many reasons. The backing code in `CacheSeed.cpp` still exists in the repo if you need to use it for your own application.

**Reply by christ0phd** [reply]
Hey, Glen, yes I've rad that it was intentionally removed from the build long ago. So I was asking, what would be the currently intended way to pre-calculate the caches. I tried some steps with the suggested `osgearth_conv` but without any luck. Is this the intended way?
